content/en/docs/Policies/policies.md

```diff
@@ -90,32 +90,40 @@ A successful policy will trigger the following output:
 
 ### The Rule JSON file
 
-The rule files follow this naming convention: `<cloud-provider>.<resource-type>.<rule-category>.<severity>.<next-available-rule-number>.json`
+The rule files follow this naming convention: `AC_<policy_type>_<next_available_rule_number>.json` where `<policy_type` is the upper case of any supported policy types by terrascan. These can be fetched from the `terrascan scan -h` help menu.
```

Comment:
The `terrascan scan` help menu doesn't show the naming convention.

Reply:
What I meant is that you can obtain the <policy_type> values by doing `terrascan scan -h | grep "policy-type"`. I'll reword this so that it's clear.